src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.container {
+  @include on1024;
+  padding: 50px 128px 72px;
+  display: grid;
+  grid-template-columns: 200px 200px 200px 200px;
+  grid-template-rows: 408px 408px;
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
```

Comment:
define transition property as well (it's better for performance than `all` default value)

Reply:
Agree
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Reply:
also, make a variable for transitionDuration